File: 251002/Urbanovich/Lab1.Core/Models/Creator.cs
using System.Reflection.Emit;

namespace Lab1.Core.Models
{
    public class Creator
    {
        public string Login { get; } = string.Empty;
        public string Password { get; } = string.Empty;
        public string FirstName { get; } = string.Empty;
        public string LastName { get; } = string.Empty;
        private Creator(string login, string password, string firstName, string lastName) => (Login, Password, FirstName, LastName) = (login, password, firstName, lastName);
        public static Creator Construct(string login, string password, string firstName, string lastName)
        {
            return new Creator(login, password, firstName, lastName);
        }
    }
}

File: 251003/Molchan/Publisher/src/main/java/by/molchan/Publisher/models/Creator.java
package by.molchan.Publisher.models;

import jakarta.persistence.*;

import java.io.Serializable;


@Entity
@Table(name = "tbl_creator")
public class Creator implements Serializable {
    @Id
    @Column(name = "id")
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private long id;

    @Column(name = "login", unique = true)
    private String login;

    @Column(name = "password")
    private String password;

    @Column(name = "firstname")
    private String firstname;

    @Column(name = "lastname")
    private String lastname;

    public String getFirstname() {
        return firstname;
    }

    public void setFirstname(String firstname) {
        this.firstname = firstname;
    }

    public long getId() {
        return id;
    }

    public void setId(long id) {
        this.id = id;
    }

    public String getLastname() {
        return lastname;
    }

    public void setLastname(String lastname) {
        this.lastname = lastname;
    }

    public String getLogin() {
        return login;
    }

    public void setLogin(String login) {
        this.login = login;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }
}
